testing/src/test/java/io/stargate/it/BaseOsgiIntegrationTest.java

```diff
@@ -41,8 +50,30 @@
     }
   }
 
+  @BeforeAll
+  public static void beforeAll(StargateEnvironmentInfo stargate) {
+    STARGATE_ADDRESSES =
+        stargate.nodes().stream()
+            .map(
+                n -> {
+                  try {
+                    return InetAddress.getByName(n.seedAddress());
+                  } catch (UnknownHostException e) {
+                    throw new RuntimeException(e);
+                  }
+                })
+            .collect(Collectors.toList());
+  }
```

Comment:
Pulled up from `JavaDriverTestBase` (which is gone now). It's a bit awkward but its used in a few places and I don't want every test to reimplement it. Maybe we could just turn it into a method on `StargateEnvironmentInfo` and tests that need it can inject it.

Reply:
Nevermind, I've fixed that.